app/src/androidTest/java/com/github/polybooks/database/FBSaleDatabaseTest.kt

```diff
@@ -68,7 +68,7 @@ class FBSaleDatabaseTest {
     fun t_searchByTitle() {
```

Comment:
I know I'm the one who ignored most of the tests, but as I expected, once you mocked the db, you could re-enable the tests. You missed this ignore then.